service/gcs/prot/protocol.go

```diff
@@ -414,6 +421,15 @@ type MappedVirtualDisk struct {
 	ReadOnly          bool  `json:",omitempty"`
 }
 
+// MappedDirectory represents a directory on the host which is mapped to a
+// directory on the guest through a technology such as Plan9.
+type MappedDirectory struct {
+	ContainerPath     string
+	CreateInUtilityVM bool   `json:",omitempty"`
```

Comment:
You're right, we could technically leave it out with about the same functionality. I'm keeping it in though because a) we'll probably want to support creating mapped directories and disks inside a container at some point, and b) if we don't produce an error when CreateInUtilityVM is false, then a client on the host side could pass in false, not receive an error, and then go on operating under the assumption the mount they created is actually inside the container, which is behavior we probably don't want.